orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -89,6 +64,7 @@ def __init__(
 
         self._logger = logger
 
+        self._rt_options = RuntimeOptions(self._logger)
```

Comment:
```suggestion
        self._runtime_options = RuntimeOptions(self._logger)
```
Would prefer full words instead of abbreviations (rt -> runtime)